src/Prods.inc.php

```diff
@@ -1 +1,3 @@
-
+<?php
+require_once("autoload.inc.php");
```

Comment:
What's your reasoning of preferring this loading method over composers autoload method? Why do you want both methods to remain working?

Reply:
nothing in particular. it worked with yoda, without it i had to do some more yoda changes. but i'm no php expert